.vscode/settings.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Hey @dafna-bashan, this file seems not relevant (probably some local vscode colors config)

Reply:
Hey @ChengaDev, thanks for the review! I deleted this folder.